server/views/pages/changeUsedFor/details.njk

```diff
@@ -34,10 +35,10 @@
   <div class="govuk-width-container">
     <div class="govuk-grid-row govuk-!-margin-bottom-2">
       <div class="govuk-grid-column-two-thirds">
-        <span class="govuk-caption-m" data-qa="cell-name"> {{ locationName }}</span>
+        <span class="govuk-caption-m" data-qa="cell-name">{{ locationType }} {{ locationName }}</span>
         <h1 class="govuk-heading-l">Change what the location is used for</h1>
         {{ govukWarningText({
-          text: 'This change will apply to all normal accommodation locations in ' + locationName + '.',
+          text: 'This change will apply to all normal accommodation locations in ' + locationType + ' ' + locationName + '.',
```

Comment:
the way you have this means that if the wing has a local name of `Wing A`, the text will read `Wing Wing A`, the locations decorator adds `displayName` to location which should avoid that

see `deactivateTemporary/details.njk`, you'll need to pass `true` into your `populateLocation()` in `server/routes/changeUsedFor/index.ts:12` and then any usages of the decorated fields like `location.usedFor` will need to be `location.raw.usedFor` instead

Reply:
Will create seperate PR for that fix :)